### PR TITLE
Different "htmlValue" value depending on browser

### PR DESCRIPTION
**Pull request by Mihaiii**
For Issue #78 .

In "content-wrapped-tinymce" there were some workarounds that came with the element.
They are related to the fact that `this.MCEInstance.getContent();` (this is the same as `this.MCEInstance.getContent({ format : 'html' } );`) doesn't work in Firefox.
The thing is that `this.MCEInstance.getContent({format: 'raw'});` works and because the workarounds might be a source of different problems in different situations, I decided to remove them.
I couldn't manage to find a situation in which those workarounds would be prefered over `getContent({ format : 'raw' } )`.

With `format: raw`, the output has those `<br data-mce-bogus="1">`. There is a comment in the source code of tinymce related to them:
```"
// Padd empty content in Gecko and Safari. Commands will otherwise fail on the content
// It will also be impossible to place the caret in the editor unless there is a BR element present
```

After trying different options at init, I decided to just remove them( `<br data-mce-bogus="1">`) after the "raw value" comes. I'm perfectly fine with that solution. In Chrome, for `getContent` with "html" format, they come already removed.

### CODE REVIEW

**Comment by Mihaiii** [comment]
This PR/commit also contains the changes made in PR #81 . 
Sorry for that! (I put them there so I can test easier and forgot to remove them).

**Comment by Mihaiii** [comment]
Noticed few minutes ago that when the tinymce instance is removed, the `remove `method calls `save` and `save` uses `getContent()` which eventually throws an error ofc.

I will probably update an existing PR instead of making a new issue/PR for this.
I also tested in `develop` branch.
Works fine in Chrome.

detached2

File: src/Content/wwwroot/Content/elements/content-wrapped-tinymce/content-wrapped-tinymce.html [hunk 1]
```diff
@@ -67,6 +51,9 @@
                     forced_root_block : "p",
                     format: 'raw',
                     remove_linebreaks: false,
+                    convert_newlines_to_brs: false,
+                    remove_redundant_brs: true,
+                    remove_trailing_brs: true,
```
Comment (Mihaiii) [inline src/Content/wwwroot/Content/elements/content-wrapped-tinymce/content-wrapped-tinymce.html]: Not really, but that's just only the conclusion I got after some tests. The idea was that if they are used in some situations, that's the desired option so I left them in the source code. Do you want me to remove them?

Comment (Mackiovello) [inline src/Content/wwwroot/Content/elements/content-wrapped-tinymce/content-wrapped-tinymce.html]: If I'm not mistaken, they were added in https://github.com/StarcounterApps/Content/pull/82/commits/d68789a13b9a0d83633daece616d39937c72528a, so it should be safe to remove

**Pull request merged**